tenders/hvt/hvt_main.c

```diff
@@ -54,6 +54,8 @@ static void setup_modules(struct hvt *hvt, struct mft *mft)
 
     bool fail = false;
     for (unsigned i = 0; i != mft->entries; i++) {
```

Comment:
I guess these loops can now also start at `1`?

Reply:
I added

        if (mft->e[i].type >= MFT_RESERVED_ZERO)
            continue;

in the loop body, it's more explicit than a `1`-based loop.